python/tvm/meta_schedule/testing/torchbench/util.py

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+import sys
+from dataclasses import dataclass
+
+candidates = [
+    "torchdynamo",
+    "../torchdynamo",
+    "../../torchdynamo",
+]
+for library_dir in candidates:
+    if os.path.exists(f"{library_dir}/benchmarks"):
+        break
+else:
+    raise RuntimeError(
+        f"Cannot find directory for torchdynamo. You need to clone https://github.com/pytorch/torchdynamo to the parent directory of cwd."
+    )
+
+sys.path.append(library_dir)
+sys.path.append(f"{library_dir}/benchmarks")
+from common import same, timed
+from torchbench import TorchBenchmarkRunner
+
+
+@dataclass
+class RunnerArgs:
```

Comment:
Move this into `load_torchdynamo_benchmark_runner`, and add some parameters to `load_torchdynamo_benchmark_runner`